google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -1823,6 +1823,12 @@ def self.new_reference project_id, dataset_id, service
         # @param [Boolean] autocreate Specifies whether the method should create
         #   a new table with the given `table_id`, if no table is found for
         #   `table_id`. The default value is false.
+        # @param [Array<String>] insert_ids A unique ID for each row. BigQuery
```

Comment:
Would you be okay listing the `insert_ids` as the first optional argument, just after the `rows` positional argument?

Reply:
And move it in the method signatures as well?

Reply:
Yes, I think it would be best to have the documentation match the method declaration.